Melanzana.CodeSign/Bundle.cs

```diff
@@ -95,27 +105,23 @@ public void AddResourceRules(ResourceBuilder builder, bool useV2Rules = true)
                     builder.AddRule(new ResourceRule($"^{resourcePrefix}") { Weight = 20 });
                 }
 
-                builder.AddRule(new ResourceRule(".*\\.dSYM($|/)") { Weight = 11 });
-
                 // Exclude specific files:
                 builder.AddRule(new ResourceRule("^Info\\.plist$") { IsOmitted = true, Weight = 20 });
                 builder.AddRule(new ResourceRule("^PkgInfo$") { IsOmitted = true, Weight = 20 });
 
                 // Include specific files:
                 builder.AddRule(new ResourceRule("^embedded\\.provisionprofile$") { Weight = 20 });
-                builder.AddRule(new ResourceRule("^version.plist$") { Weight = 20 });
-
-                builder.AddRule(new ResourceRule("^(.*/)?\\.DS_Store$") { IsOmitted = true, Weight = 2000 });
+                builder.AddRule(new ResourceRule("^version\\.plist$") { Weight = 20 });
             }
             else
             {
-                builder.AddRule(new ResourceRule("^version.plist$"));
                 builder.AddRule(new ResourceRule(hasResources ? $"^{resourcePrefix}" : "^.*"));
-            }
 
-            builder.AddRule(new ResourceRule($"^{resourcePrefix}.*\\.lproj/") { IsOptional = true, Weight = 1000 });
-            builder.AddRule(new ResourceRule($"^{resourcePrefix}Base\\.lproj/") { Weight = 1010 });
-            builder.AddRule(new ResourceRule($"^{resourcePrefix}.*\\.lproj/locversion.plist$") { IsOmitted = true, Weight = 1100 });
+                builder.AddRule(new ResourceRule($"^{resourcePrefix}.*\\.lproj/") { IsOptional = true, Weight = 1000 });
+                builder.AddRule(new ResourceRule($"^{resourcePrefix}.*\\.lproj/locversion.plist$") { IsOmitted = true, Weight = 1100 });
+                builder.AddRule(new ResourceRule($"^{resourcePrefix}Base\\.lproj/") { Weight = 1010 });
+                builder.AddRule(new ResourceRule("^version.plist$"));
```

Comment:
This one is missing the `version.plist` -> `version\\.plist` fix.

Reply:
```suggestion
                builder.AddRule(new ResourceRule("^version\\.plist$"));
```

Reply:
Turns out the bug is present in Apple implementation too, so I just added comment.